lib/swaggerValidator/trafficValidator.ts

```diff
@@ -44,9 +45,11 @@ export class TrafficValidator {
   private trafficPath: string;
   private loader?: LiveValidatorLoader;
   private trafficOperation: Map<string, string[]> = new Map<string, string[]>();
+  private validationFailOperations: Map<string, string[]> = new Map<string, string[]>();
+  private coverageData: Map<string, number> = new Map<string, number>();
   public operationSpecMapper: Map<string, string[]> = new Map<string, string[]>();
-  public coverageResult: Map<string, number> = new Map<string, number>();
-  public coverageData: OperationCoverageInfo[] = [];
+  public coverageResult: OperationCoverageInfo[] = [];
```

Comment:
My bad, can you rename coverageResult to operationCoverageResult, rename undefinedResult to operationUndefinedResult  for better understanding?